pkg/manager/manager.go

```diff
@@ -458,8 +427,12 @@ func (s *singleton) GetSystemStatus() *models.SystemStatus {
 }
 
 // Shutdown gracefully stops the manager
-func (s *singleton) Shutdown() error {
+func (s *singleton) Shutdown(code int) {
```

Comment:
Do we really need to force the exit code? 
If everything is ok it should be `0` otherwise `1`
In this case the only thing  we do is close the db, so if the err != nil we should then os.Exit(1) , not only log the error

Reply:
**Not intended for the scope of this PR** 
It seems that the only way to reach the shutdown before this PR was for the user to either ctrl-c in the stash terminal or kill the stash process. With this PR this can be done  easily from the systray. We should find a proper way to shutdown  and wait for the manager (eg context cancelation) so that any running tasks are properly terminated before attempting to close the DB.

Reply:
Yeah, that's been on the to-do list a while.

I made it take an exit code so that a function that's bailing out can drop a code in. I can probably do an override if the parameter is 0 and it runs into an error closing the database, return a 1 instead.

Reply:
Yeah that  sounds good.

Reply:
Added a check to see return error code if necessary